configure.ac

```diff
@@ -609,6 +609,28 @@ AS_IF([test "$have_ze" = "1" && test x"$with_ze" != x"yes"],
       [CPPFLAGS="$CPPFLAGS $ze_CPPFLAGS"
        LDFLAGS="$LDFLAGS $ze_LDFLAGS"])
 
+dnl Check for AWS Neuron runtime library for Neuron device support
+AC_ARG_WITH([neuron],
+	AS_HELP_STRING([--with-neuron=DIR],
+		       [Provide path to where the Neuron headers are installed.]),
+	[], [])
+
+_FI_CHECK_PACKAGE_HEADER([neuron], [nrt/nrt.h], [$with_neuron],
+	[
+		AS_IF([test "$freebsd" = "0"], [
+		    AC_CHECK_LIB(dl, dlopen, [],
+			[AC_MSG_ERROR([dlopen not found.  libfabric requires libdl.])])
```

Comment:
Doesn't the check for dlopen already exist?

Reply:
It does, but looks like it is optional (have to pass `--with-dlopen` to force the check).